<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>StakePage</class>
    <widget class="QWidget" name="StakePage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>744</width>
                <height>450</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
                <widget class="QFrame" name="frameStakeInfo">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <layout class="QGridLayout" name="gridLayout_2">
                                <item row="0" column="0">
                                    <widget class="QLabel" name="labelAssetsText">
                                        <property name="text">
                                            <string>Staking assets</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="0">
                                    <widget class="QLabel" name="labelAssets">
                                        <property name="font">
                                            <font>
                                                <weight>50</weight>
                                                <bold>false</bold>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>IBeamCursor</cursorShape>
                                        </property>
                                        <property name="toolTip">
                                            <string>Your current spendable balance</string>
                                        </property>
                                        <property name="text">
                                            <string notr="true">0.000 000 00 HYDRA</string>
                                        </property>
                                        <property name="textInteractionFlags">
                                            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <spacer name="horizontalSpacer_2">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="1" column="2">
                                    <widget class="QSwitchControl" name="checkStake">
                                        <property name="enabled">
                                            <bool>false</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="3">
                                    <widget class="QLabel" name="labelCheckStakeText">
                                        <property name="text">
                                            <string>Staking on/off</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="0">
                                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                                        <item>
                                            <widget class="QLabel" name="labelStakeText">
                                                <property name="text">
                                                    <string>Stake:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="labelStake">
                                                <property name="font">
                                                    <font>
                                                        <weight>50</weight>
                                                        <bold>false</bold>
                                                    </font>
                                                </property>
                                                <property name="text">
                                                    <string>0.000 000 00 HYDRA</string>
                                                </property>
                                                <property name="alignment">
                                                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="Line" name="hLine">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QGridLayout" name="gridLayout">
                                <item row="0" column="0">
                                    <layout class="QHBoxLayout" name="horizontalLayout">
                                        <item>
                                            <widget class="QLabel" name="labelHeightText">
                                                <property name="text">
                                                    <string>Block height:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="labelHeight">
                                                <property name="text">
                                                    <string>0</string>
                                                </property>
                                                <property name="alignment">
                                                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                <item row="0" column="1">
                                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                                        <item>
                                            <widget class="QLabel" name="labelRewardText">
                                                <property name="text">
                                                    <string>Block reward:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="labelReward">
                                                <property name="text">
                                                    <string>0.4 HYDRA/Block</string>
                                                </property>
                                                <property name="alignment">
                                                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                <item row="0" column="2" rowspan="2">
                                    <spacer name="horizontalSpacer">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="1" column="0">
                                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                                        <item>
                                            <widget class="QLabel" name="labelWeightText">
                                                <property name="text">
                                                    <string>Network weight:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="labelWeight">
                                                <property name="text">
                                                    <string>0</string>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                <item row="1" column="1">
                                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                                        <item>
                                            <widget class="QLabel" name="labelROIText">
                                                <property name="text">
                                                    <string>Expected annualized ROI:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="labelROI">
                                                <property name="text">
                                                    <string>7.9 %</string>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>30</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QFrame" name="frameStakeRecords">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QLabel" name="labelRecordsText">
                                <property name="text">
                                    <string>Stake records</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
        <widget class="QLabel" name="labelAlerts">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>16</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="visible">
                <bool>false</bool>
            </property>
            <property name="styleSheet">
                <string notr="true">QLabel { background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #F0D0A0, stop:1 #F8D488); color:#000000; }</string>
            </property>
            <property name="wordWrap">
                <bool>true</bool>
            </property>
            <property name="margin">
                <number>3</number>
            </property>
            <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
            </property>
        </widget>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QSwitchControl</class>
            <extends>QCheckBox</extends>
            <header>qt/qswitchcontrol.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>